docs/src/components/landingBlocks/BuiltByBlock.tsx

```diff
@@ -44,11 +41,8 @@ const BuiltByBlock = ({ content }: Props) => {
     <Section>
       <Container>
         <Title>{content.title}</Title>
-        <Image
-          src={NomicFoundationLogo}
-          alt="Nomic Foundation logo"
-          quality={100}
-        />
+        {/* eslint-disable-next-line */}
+        <img src={content.imageUrl} alt="Nomic Foundation logo" />
```

Comment:
what the reason for switching from <Image> to <img>? How does it affect performance? Let's discuss it on a sync first